api/tests.py

```diff
@@ -314,6 +326,41 @@ def test_cannot_change_meta_tag_color(self):
         meta_tag = puzzle.tags.get(name=meta.name)
         self.assertEqual(meta_tag.color, PuzzleTag.BLACK)
 
+    def test_empty_custom_tags_deletion(self):
```

Comment:
moved these two tests from puzzles/test.py, since they're testing code in api/views.py and not stuff in puzzles/